.ci/appveyor.yml

```diff
@@ -39,6 +39,13 @@ install:
   - "python --version"
   - "python -c \"import struct; print(struct.calcsize('P') * 8)\""
 
+  - "%CMD_IN_ENV% pip install appdirs"
+  - git clone https://github.com/coala-analyzer/coala coala-manuall-branch-inst
+  - cd coala-manuall-branch-inst
```

Comment:
May want to add `coala-manuall-branch-inst` tot he ignored folders of pytest
The tests in this folder will get run too